screens/TranscriptionScreen.tsx

```diff
@@ -0,0 +1,69 @@
+import { StackScreenProps } from "@react-navigation/stack";
+import React, { useState, useEffect } from "react";
+import { StyleSheet, View, Text, Button, Pressable } from "react-native";
+import { RootStackProps } from "../App";
+import { base64ImageToTensor } from "../util/image";
+import * as tf from "@tensorflow/tfjs";
+import "@tensorflow/tfjs-react-native";
+import * as cocoSsd from "@tensorflow-models/coco-ssd";
+import * as FileSystem from "expo-file-system";
+
+type TranscriptionProps = StackScreenProps<RootStackProps, "Transcription">;
+
+const TranscriptionScreen = ({ navigation, route }: TranscriptionProps): JSX.Element => {
+	const [objectDetectionPreidictions, setObjectDetectionPreidictions] =
+		useState<cocoSsd.DetectedObject[]>();
+	const [status, setStatus] = useState<string>("Waiting for image");
+	const [showFilePath, setShowFilePath] = useState<boolean>(false);
+
+	useEffect(() => {
+		const handleImage = async () => {
+			setStatus("Reading...");
+			const buffer = await FileSystem.readAsStringAsync(route.params.latestImagePath, {
+				encoding: FileSystem.EncodingType.Base64,
+			});
+			setStatus("Processing...");
+			const imageTensor: tf.Tensor3D = base64ImageToTensor(buffer);
+
+			setStatus("Detecting objects...");
+			const predictions = await route.params.objectModel?.detect(imageTensor);
+			setObjectDetectionPreidictions(predictions);
+			setStatus(`Objects: ${JSON.stringify(objectDetectionPreidictions)}`);
+		};
```

Comment:
I would prefer if `TranscriptionView/Screen` just did UI stuff, and not the actual image processing.

Reply:
Where would you like me to move it? Ideally, it does the processing in the TranscriptionView so we could potentially display a progress bad, or maybe a prettied-up logs if it fits the design.

Reply:
`App.tsx`, we can easily have a modal or something for UI during processing instead of doing it in the transcription screen.